get_current_tidepool_repos.sh

```diff
@@ -10,10 +10,10 @@ require()
 require git "Visit git-scm.com to get it."
 require node "Visit nodejs.org to get it."
 require npm "Visit npmjs.org to get it."
-require gulp "Visit gulpjs.com to get it."
-require mocha "Visit http://visionmedia.github.io/mocha/ to get it."
+require gulp "Use 'npm install --global gulp' to get it."
+require mocha "Use 'npm install --global mocha' to get it."
 require cc "On a Mac, install XCode and its command line tools."
-require mongod "Visit mongodb.org to get it."
+require mongod "Use [brew update] and 'brew install mongodb' to get it."
```

Comment:
There's scoop, chocolatey and others.  But I didn't add the dependency/suggestion to use brew install because that was already in line 18.  I can make it clearer though.

One more note; i originally followed the mongodb site instructions for download and unpacking manually, but it didn't install in a way that the tidepool apps could automatically use it.  Whereas homebrew did the right thing of installing it and getting it ready to go.